config/initializers/govuk_errors.rb

```diff
@@ -1,3 +0,0 @@
-GovukError.configure do |config|
-  config.excluded_exceptions << 'AssetManager::AttachmentReplacementIdUpdater::AssetNotFound'
```

Comment:
Could you clarify in the commit message why we're no longer excluding this?

Reply:
Done:

> I've also removed ignoring the old 
`AssetManager::AttachmentReplacementIdUpdater::AssetNotFound`
exception in Sentry as this exception has been merged together
with another and since we don't currently ignore the one it was
merged into it would be best to not start hiding more than we do
currently. If we decided that we don't need to see this exception
(that I suspect is the case) we can add it to the ignore list in
a later PR.